sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -406,6 +416,24 @@ public static void checkForUpdate() {
         getInstance().instanceCheckForUpdate();
     }
 
+    /**
+     * Disable automatic check for update before the service starts.
+     */
+    public static void disableAutomaticCheckForUpdate() {
```

Comment:
move it next to other static methods

Reply:
It's already next to the other static methods, though?